pytest_playwright/pytest_playwright.py

```diff
@@ -203,6 +207,11 @@ def browser_channel(pytestconfig: Any) -> Optional[str]:
     return pytestconfig.getoption("--browser-channel")
 
 
+@pytest.fixture(scope="session")
+def device(pytestconfig: Any) -> Optional[str]:
```

Comment:
why to define the device as a fixture? We read now the option in both places and the one in the device fixture is probably unused.